bin/www

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+var app = require('../app');
+var debug = require('debug')('aarons-blog:server');
+var http = require('http');
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+var port = normalizePort(process.env.PORT || '3000');
```

Comment:
I think it would help us if we could see what port the app is booting on when we boot the app.

Something as simple as:
```
console.log("App is booting on http://localhost:" + port);
```